README.md

```diff
@@ -1,42 +1,39 @@
 # Creative Commons Licenses for GitHub Projects
 
-In this repo you can find easy ways for applying Creative Commons Licenses on
-Github repositories through Markdown language.
+[![Deutsch](https://img.shields.io/badge/translation-DE-red)](de/README.md)
```

Comment:
Nice badge! I would point towards the `de` directory instead to the `README.md` itself. The readme will be shown, but also the LICENSE files.
```suggestion
[![Deutsch](https://img.shields.io/badge/translation-DE-red)](de/)
```

Reply:
Also thought about it but you wrote a few comments before that you want to link to READMEs.